rasa_core/training/interactive.py

```diff
@@ -394,6 +398,8 @@ def add_user_cell(data, cell):
     for idx, evt in enumerate(evts):
         if evt.get("event") == "action":
             bot_column.append(colored(evt['name'], 'autocyan'))
+            if evt['confidence']:
```

Comment:
should this rather be `... is not None` - otherwise this won't print `0.00` confidence scores, or is that intended?

Reply:
Good catch!  Added